src/modules/Dashboard/features/Managers/Managers.utils.tsx

```diff
@@ -52,26 +56,57 @@ export const getHeadCells = (sortableColumns?: string[], appletId?: string): Hea
 };
 
 export const getManagerActions = (
-  { removeAccessAction, editAccessAction }: ManagersActions,
+  actions: ManagersActions,
   manager: Manager,
-) => {
+): MenuItem<Manager>[] => {
   const { t } = i18n;
 
-  return [
+  const menuItems: MenuItem<Manager>[] = [];
+
+  if (manager.status === 'pending') {
+    menuItems.push(
+      {
+        type: MenuItemType.Info,
+        title: `${t('invitationDate')}: ${format(
+          new Date(manager.createdAt),
+          `${DateFormats.MonthDayYear} '${t('lowercaseAt')}' ${DateFormats.Time}`,
+        )}`,
+      },
+      { type: MenuItemType.Divider },
+      {
+        icon: <Svg id="duplicate" width={24} height={24} />,
+        action: actions.copyEmailAddressAction,
+        title: t('copyEmailAddress'),
+        context: manager,
+      },
+      { type: MenuItemType.Divider },
+      {
+        icon: <Svg id="format-link" width={24} height={24} />,
+        action: actions.copyInvitationLinkAction,
+        title: t('copyInvitationLink'),
+        context: manager,
+      },
+      { type: MenuItemType.Divider },
+    );
+  }
+
+  menuItems.push(
     {
       icon: <Svg id="edit-user" />,
-      action: editAccessAction,
-      title: t('editAccess'),
+      action: actions.editTeamMemberAction,
+      title: t('editTeamMember'),
       context: manager,
       'data-testid': 'dashboard-managers-edit-user',
     },
     {
       icon: <Svg id="remove-access" />,
-      action: removeAccessAction,
-      title: t('removeAccess'),
+      action: actions.removeTeamMemberAction,
+      title: t('removeTeamMember'),
       context: manager,
       customItemColor: variables.palette.dark_error_container,
       'data-testid': 'dashboard-managers-remove-access',
     },
-  ];
+  );
```

Comment:
I intentionally didn't remove those because of the lack of the cancel invitation action in this ticket. I got some [feedback](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1769#pullrequestreview-2091138461) to this end when I implemented this for the participant menu, and I figured it would make sense to leave removing that until when the cancel action is being implemented.

However, it seems that removing a team member functions differently than removing a participant because all participants have a subject, whereas invited team members have no access until they accept the invitation, and so the request fails.

I'll remove the menu items because they have no functionality anyway